privacyidea/api/validate.py

```diff
@@ -251,7 +254,13 @@ def check():
         request
     :param state: The state ID for a response to a challenge request
 
-    :return: a json result with a boolean "result": true
+    :return:
+        | Value depends on the endpoint used:
+        | For /validate/samlcheck: a map with the at least the following user
+            attributes: ``username``, ``realm``, ``resolver``, ``email``
+            ``surname``, ``givenname``, ``mobile``, ``phone``
+        | For /validate/radiuscheck: no response body
+        | For /validate/check: a json boolean
```

Comment:
Please keep ths specific:

     For  /validate/check a json response with booleans in "result"->"status" and"result"->"value"